.gitignore

```diff
@@ -26,7 +26,7 @@ yarn-error.log*
 .pnpm-debug.log*
 
 # local env files
-.env
+.env*
```

Comment:
This change will remove `.env.example` from being in the repo for users to view it as example.